UI-Session-4-Assignment/index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Home</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous" />
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <!-- header for all web pages of this site contain logo-image ,menu and social links of facebook instagram and google plus-->
+    <header id="header">
+        <!--company logo-->
+        <img class="logo" src="images\logo.png" alt="Logo image">
```

Comment:
Alignment of logo image is not same as the layout.

Reply:
Done